utils/sl/fake_scons/SCons/Warnings/__init__.py

```diff
@@ -19,12 +19,19 @@
 # SOFTWARE.
 """Fake scons environment shutting up pylint on SCons files"""
 
+
 def warningAsException(flag=1):
     """Fake warningAsException"""
     return flag
 
+
 def enableWarningClass(_name):
     """Fake enableWarningClass"""
 
+
 class MissingSConscriptWarning(Exception):
     """Fake MissingSConscriptWarning"""
+
+
+class Warning():
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'Warning'
(pylint-redefined-builtin) Redefining built-in 'Warning'
(pylint-too-few-public-methods) Too few public methods (0/2)